<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_src">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_chargement">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Charger OBJ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_vertex_src">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>ajouter sommet de contrôle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_spherizationSrc">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Spherization</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_despherizationSrc">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Despherization</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="MeshViewerWidget" name="displayWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_tgt">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_chargement_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Charger OBJ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_vertex_tgt">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>ajouter sommet de contrôle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_spherizationTgt">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Spherization</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_despherizationTgt">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Despherization</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="MeshViewerWidget" name="displayWidget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="MeshViewerWidget" name="displayWidget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_spherization">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Spherization</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_despherization">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Despherization</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_superMesh">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>SuperMesh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MeshViewerWidget</class>
   <extends>QWidget</extends>
   <header location="global">meshviewerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
